server/command/command.go

```diff
@@ -49,16 +49,75 @@ func NewHandler(args *model.CommandArgs, conf *config.Config, a splunk.Splunk) H
 
 // GetSlashCommand returns command to register
 func GetSlashCommand() *model.Command {
+	splunk := model.NewAutocompleteData(
+		slashCommandName, "[alert|auth|help|log]", "connect to and interact with splunk.")
+	addSubCommands(splunk, true)
+
 	return &model.Command{
 		Trigger:          slashCommandName,
 		DisplayName:      slashCommandName,
 		Description:      pluginDescription,
 		AutoComplete:     true,
+		AutocompleteData: splunk,
 		AutoCompleteDesc: autoCompleteDescription,
 		AutoCompleteHint: autoCompleteHint,
 	}
 }
 
+func addSubCommands(splunk *model.AutocompleteData, optInstance bool) {
```

Comment:
It doesn't look like `optInstance` is being used anywhere so we should remove it. This was copied from the Jira plugin, which uses that pattern for its own use cases.

Reply:
Sure